tests/ResourceSyncingTest.php

```diff
@@ -284,14 +284,10 @@
 
     tenancy()->end();
 
-    $centralUser = CentralUserProvidingMixture::whereGlobalId('acmey')->first();
+    $centralUser = CentralUserProvidingMixture::whereGlobalId('acmey')->first()->only(['name', 'email', 'password', 'role']);
     expect($resourceUser->getSyncedCreationAttributes())->toBeNull();
 
-    $centralUser = $centralUser->toArray();
-    $resourceUser = $resourceUser->toArray();
-    unset($centralUser['id']);
-    unset($resourceUser['id']);
-
+    $resourceUser = $resourceUser->only(['name', 'email', 'password', 'role']);
```

Comment:
What's the reason for this change?

Reply:
Just cleaning/improvements. I used the same technique in newly added tests and thought I should refactor them too. Changes in `ResourceSyncingTest.php` are not related to PR/ or needed.

Reply:
I don't think this approach is good. Below, we are asserting this:
```php
// Assert central user created as 1:1 copy of resource model except "id"
expect($centralUser)->toBe($resourceUser);
```

Which implies that the models should be ENTIRELY identical (except the id for obvious reasons).

Selecting only a few columns to compare defeats the point here — we want to check that the models are completely identical.

I think both in this test, and in the tests you added, you should be using the `unset()` approach because it makes it clear that we assert that the models are the same.

Using `only()` *might* be the same thing here, **but it's not obvious to people who read the test**. They'd need to check the migrations to see that. So it's better to make it explicit in the code.

Reply:
> They'd need to check the migrations to see that. So it's better to make it explicit in the code.

Makes sense. I'll update.